src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -78,6 +74,8 @@ private static FeedSource getFeedSource(Request req, Response res) {
         }
         Collection<FeedSource> projectFeedSources = project.retrieveProjectFeedSources();
         for (FeedSource source: projectFeedSources) {
+            // Verify user permissions before returning source
+            source = checkFeedSourcePermissions(req, source, Actions.VIEW);
```

Comment:
Isn't this already handled with some code below?

Reply:
The issue was removing admin-only labels, but I've extracted this code so these lines are now removed.